docs/docusaurus/docs/core/introduction/community_resources.md

```diff
@@ -1,8 +1,22 @@
 ---
-title: Contribute
-id: contributing
+title: 'Community Resources'
 ---
+import GxCloudAdvert from '/static/docs/_static_components/_gx_cloud_advert.md'
 
+Great Expectations (GX) is committed to supporting and growing the GX community. It’s not enough to build a great tool. GX wants to build a great community as well.
+
+Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and GX is dedicated to making you feel welcome.
+
+## Contact Great Expectations
+Join the GX [public Slack channel](https://greatexpectations.io/slack). Before you post for the first time, review the [Slack Guidelines](https://discourse.greatexpectations.io/t/slack-guidelines/1195).
+
+## Ask a question
+Slack is good for that, too: [join Slack](https://greatexpectations.io/slack) and read [How to write a good question in Slack](https://github.com/great-expectations/great_expectations/discussions/4951). You can also use [GitHub Discussions](https://github.com/great-expectations/great_expectations/discussions/4951).
+
+## File a bug report or feature request
+If you have bugfix or feature request, see [upvote an existing issue](https://github.com/great-expectations/great_expectations/issues) or [open a new issue](https://github.com/great-expectations/great_expectations/issues/new).
```

Comment:
I believe the plan is to open this back up after we launch 1.0, but it would be worth checking in with Timber to verify.

Reply:
Are we actually able to upvote GH issues? The link just takes me to the GitHub open issues page, I'm not sure users can upvote (my understanding is this is what the future bounty board will enable).

Reply:
I'll check with Timber and comment out the references to github for now, then.